<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>43</y>
      <width>461</width>
      <height>421</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableView" name="singleView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>461</width>
       <height>421</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="lower">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>480</y>
      <width>474</width>
      <height>132</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>460</width>
      <height>130</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>11</y>
       <width>471</width>
       <height>112</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="intextLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>I/E</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="intextChoice">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
          <item>
           <property name="text">
            <string>I</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>E</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="bankLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Bank</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="bankChoice">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
          <item>
           <property name="text">
            <string>A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>C</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>D</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>a</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>b</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>c</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>d</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="slotLabel">
          <property name="text">
           <string>Slot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="slotChoice">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,0,0">
        <item>
         <widget class="QLabel" name="portLabel">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="portChoice"/>
        </item>
        <item>
         <widget class="QToolButton" name="refreshPortsButton">
          <property name="toolTip">
           <string>Refresh port list</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="view-refresh">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="channelLAbel">
          <property name="text">
           <string>Channel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="channelChoice">
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>70</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>12</y>
      <width>168</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Show bank</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="filterChoice">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>All</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
